data/wordlists/tomcat_mgr_default_users.txt

```diff
@@ -1,6 +1,8 @@
 admin
 manager
 role1
+role
 root
 tomcat
 both
+QCC
```

Comment:
Missing following users from userpass.txt:

```
j2deployer
ovwebusr
cxsdk
ADMIN
xampp
```

Reply:
Done

Reply:
Verified resolved in 0305983be2d354c76a586cf9db7f1dc10fccbd01